ingest/mappings/mineral-observation.json

```diff
@@ -148,6 +142,25 @@
             "type":"string"
           }
         }
+      },
+      "mineral-specimen":{
+        "properties":{
+          "max-age":{
+            "type":"double"
+          },
+          "min-age":{
+            "type":"double"
+          },
+          "label-1":{
```

Comment:
@xgmachina @kolans do you know if there is any meaning behind label-1, label-2, label-3 in the RRUFF spreadsheets?  What is the difference between these three columns?